pkg/machine/e2e/basic_test.go

```diff
@@ -69,6 +70,37 @@ var _ = Describe("run basic podman commands", func() {
 		Expect(rmCon).To(Exit(0))
 	})
 
+	It("Volume ops", func() {
+		skipIfVmtype(define.HyperVVirt, "Hyper-V does not have functioning volume mounts yet #21036")
```

Comment:
I think this should use use the `FIXME #21036: ...` syntax so @edsantiago's scripts will catch this when he issue gets closed. But maybe he confirm on the proper syntax.

Also the message is rather confusing, the mounts work just fine. It is just that the path resolution works differently.